relay-event-normalization/src/normalize/mod.rs

```diff
@@ -479,6 +480,71 @@ fn normalize_measurements(
     }
 }
 
+/// Computes performance score measurements
+///
+/// This computes score from vital measurements, using config options to define how it is
+/// calculated.
+fn normalize_performance_score(
+    event: &mut Event,
+    performance_score_config: Option<&PerformanceScoreConfig>,
+) {
+    if let Some(performance_score_config) = performance_score_config {
+        for profile in &performance_score_config.profiles {
```

Comment:
perhaps this could be a method on `PerformanceScoreProfile`? 

example: https://gist.github.com/TBS1996/f3aa3d1125e68e6cc8d51744eba489c7

Reply:
@TBS1996 all our normalization logic lives in this crate, so I would stick to that structure here.